lib/agent/https_proxy_ocsp_agent.js

```diff
@@ -98,7 +99,33 @@ function connect(req, opts, fn)
 {
   var proxy = this.proxy;
   var agent = this;
-  Logger.getInstance().debug("Using proxy=%s for host %s", proxy.host, opts.host);
+
+  const prxPort = proxy.port ? ':' + proxy.port : '';
+  const prxUser = proxy.auth ? ' (user: ' + proxy.auth.split(':')[0] + ')' : '';
+  const prxHostWithPort = proxy.host + prxPort;
+  const envProxy = Util.getEnvProxy();
+
+  // using both the envvar and the proxyHost/proxyPort settings for proxy, this might conflict
+  if (envProxy.httpProxy &&
+      Util.removeScheme(envProxy.httpProxy).toLowerCase() != prxHostWithPort.toLowerCase()) {
+    Logger.getInstance().debug('Using both the HTTP_PROXY (%s) and the proxyHost:proxyPort (%s) settings to connect, '
```

Comment:
when it's different maybe let's log it with WARN, also in other added logs

Reply:
ok